Modules/Loadable/SubjectHierarchy/Widgets/qMRMLSubjectHierarchyModel.cxx

```diff
@@ -805,7 +808,8 @@ QFlags<Qt::ItemFlag> qMRMLSubjectHierarchyModel::subjectHierarchyItemFlags(vtkId
     }
 
   // Name and transform columns are editable
```

Comment:
This comment should be updated to be more generic

Reply:
Thank you, I've removed the comment (it wasn't very useful)